spot-electron/main.js

```diff
@@ -28,6 +29,12 @@ require('./src/client-control');
 require('./src/exit');
 require('./src/volume-control');
 
+process.env.USE_CLOUDWATCH_LOGS && awsLogger.maybeCreateLogStream()
```

Comment:
How is the Electron app going to get these env variables?

Reply:
I was thinking that the person that installs the app can set up these too.